cwms-servers/CWMS_HOME/bin/logWatcher.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
assuming this script is going to be executed, then first line should call out the interpreter for this file:
#!/bin/env python

Reply:
added